config/filter.d/sshd.conf

```diff
@@ -31,11 +29,11 @@ failregex = ^%(__prefix_line)s(?:error: PAM: )?[aA]uthentication (?:failure|erro
             ^%(__prefix_line)sUser .+ from <HOST> not allowed because a group is listed in DenyGroups\s*$
             ^%(__prefix_line)sUser .+ from <HOST> not allowed because none of user's groups are listed in AllowGroups\s*$
             ^(?P<__prefix>%(__prefix_line)s)User .+ not allowed because account is locked<SKIPLINES>(?P=__prefix)(?:error: )?Received disconnect from <HOST>: 11: .+ \[preauth\]$
-            ^(?P<__prefix>%(__prefix_line)s)Disconnecting: Too many authentication failures for .+? \[preauth\]<SKIPLINES>(?P=__prefix)(?:error: )?Connection closed by <HOST> \[preauth\]$
```

Comment:
Think about it this way..if an IP keeps connecting without attempting without trying to authenticate that is an indication of a repeated probe or a prelude to DDOS if enough IPs keep doing the same.  It would prevent legitimate access to the host.  There is no legitimate reason for an IP to keep connecting over what limits the sysadmin sets without a login attempt.